### DISCUSSION TITLE
Related Discord

### DISCUSSION THREAD

**Discussion by booniepepper** [aquaproj/aqua#2341] [Deprecated (General)] [answered]
Hey I'm from a discord server where we talk about and develop tools in the Terminals/Shells/CLIs/TUIs category, it's called [/dev/tty](https://discord.gg/QW76CzYZuH). Come join us!

Some of our members have created projects like rtx, oclif, heroku-cli, git-cliff, and more. A lot of us are also involved and have worked professionally on enterprise-scale software supply chains. We were taking a look at aqua and are really excited about a lot of what it is doing. Also, we're a polyglot community, but some members have also independently arrived at some of the same Go practices we saw in aqua source code, especially for testing. If people interested aqua are also interested in talking about Terminals/Shells/CLIs/TUIs and making the world better (supply chain, ergonomics, design choices, etc) please come join us

俺の奥さんは日本人だから日本語も話せます。（ペラペラじゃないんですけど…汗

**Comment by suzuki-shunsuke** [comment, marked as answer]
Hi, thank you for your invitation! I joined!
I'm looking forward to talking with you guys!
